server/src/thirdparty/slackAPI/controller.js

```diff
@@ -0,0 +1,82 @@
+import { sendSuccessMessage, sendFailureMessage } from '../utils/restUtil';
+import slackService from './service';
+import config from './config';
+
+const authCallBack = async (req, res) => {
+  try {
+    const { code, state } = req.query;
```

Comment:
Consider putting line breaks after a block. The try-catch block is a little unreadable

Reply:
My bad. Had async changes before, while refracting forgot to remove